ztp/gitops-subscriptions/argocd/example/policygentemplates/hub-side-templating/site-du-standard-configmap.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Is it possible to create sys-link instead of creating exact same file twice ? Same for all repeated CM files

Reply:
Unfortunately, this doesn't work with kustomize, as the files can't be in a directory higher up than the kustomization.yaml.